client/src/components/FeedbackForm/FeedbackForm.module.scss

```
@@ -0,0 +1,11 @@
+.feedback-form {
```

__Consistency__: (CSS) Use standard top-level class name for CSS modules (`.container`, discrepancies will be fixed).

```suggestion
.container {
```